Export/KL/Export.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+/*
+ * This file is part of aakb/kontrolgruppen-core-bundle.
+ *
+ * (c) 2019 ITK Development
+ *
+ * This source file is subject to the MIT license.
+ */
+
+namespace Kontrolgruppen\CoreBundle\Export\KL;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\EntityRepository;
+use Kontrolgruppen\CoreBundle\Entity\Process;
+use Kontrolgruppen\CoreBundle\Entity\ProcessStatus;
+use Kontrolgruppen\CoreBundle\Export\AbstractExport;
+use Kontrolgruppen\CoreBundle\Service\EconomyService;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+
+class Export extends AbstractExport
+{
+    protected $title = 'KL';
+
+    /** @var \Doctrine\ORM\EntityManagerInterface */
+    private $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager, EconomyService $economyService)
+    {
+        parent::__construct();
+        $this->entityManager = $entityManager;
+        $this->economyService = $economyService;
+    }
+
+    public function getParameters()
+    {
+        return parent::getParameters() + [
+                'processtatus' => [
+                    'type' => EntityType::class,
+                    'type_options' => [
+                        'label' => 'process.table.process_status',
+                        'class' => ProcessStatus::class,
+                        'query_builder' => function (EntityRepository $er) {
+                            return $er->createQueryBuilder('s')
+                                ->orderBy('s.name', 'ASC');
+                        },
+                        'required' => false,
+                        'empty_data' => null,
+                    ],
+                ],
+            ];
+    }
+
+    public function writeData()
+    {
+        $processes = $this->getProcesses();
+
+        $this->writeRow([
+            'Overføres ikke til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Overføres til KL',
+            'Kun afsluttede sager overføres til KL',
+            'Overføres ikke til KL',
+        ]);
+
+        $this->writeHeader([
+            'Sag',
```

Comment:
Translations in this file?

Reply:
@tuj, experience tells me that it doesn't really add any value to use translations in data exports. There may be very strict requirements for the labels and other stuff in the exported data.